### ISSUE TITLE
Adaptability to other exchanges API (bybit, kucoin...)

### ISSUE THREAD

**Issue by GregoireINRA** [daroczig/binancer#29]
First, let me thank you for the work you've done with binancer package.
It works great, and I'am using it a lot for research purpose.

Question is not really an issue but mostly a request/question.
(I already asked the question on stackoverflow first but I didnt get answers).

I'd like to get similar historical crypto data 'klines' for other exchanges, such as Kucoin or Bybit.
I tried to look in your functions codes but I failed on adapting your code to other exchanges API.
Would you have an idea how to proceed please ?

Thank you,
Greg

**Comment by daroczig** [comment]
Hi Greg,

Thanks a lot for your kind words.

Actually, this package was started in mind with other crypto API as well, e.g. I wrote the first few functions with support for both Binance and Huobi, but it was a burden to maintain both, and I decided to drop the latter at https://github.com/daroczig/binancer/commit/f42189dbb0ec6c30b3f21137e5314dd040feb5ee

So if anyone is stepping up to contribute and maintain wrappers for other APIs, I'm happy to take those, but it probably would make more sense to create a new package for the other networks anyway ...

In short, I will not be able to work on other API integrations in the near future, so closing this -- sorry for not being helpful.

Also, if I started this from scratch, probably would rather take a more supported Python package and wrap in in `reticulate` to be able to use in R instead of reimplementing the API client from scratch :zipper_mouth_face:
